css/style.css

```
@@ -131,59 +147,73 @@ nav li {
 
 nav a {
 	/* TODO: You could extract this colour upwards to the colours classes, as I guess we will have a hover as well. And it is always a good idea to tokenize the colours as you can easily have an overview of how many colours your project uses. */
-	color: #242f51;
+	/*DONE*/
+	color: var(--cloud_burst_blue);
 }
 
 .logo {
 	margin-right: auto;
 }
 
-/*START header_content*/
+/*START main_content*/
 
-.header_content {
-	width: 100%;
+#main_content {
 	/* TODO: When you are using multiple images as background images it is always a good idea to make them into separate lines. It will be more readable */
-	background: url(Photos/header_phone.png) 85% no-repeat, url(Photos/buildings_background_02.jpg) bottom no-repeat;
+	/*DONE*/
+	background: url(../assets/images/header_phone.png) 85% no-repeat,
+		url(../assets/images/buildings_background_02.jpg) bottom no-repeat;
```

I would even go a step further. Like this:

```suggestion
	background: 
	        url(../assets/images/header_phone.png) 85% no-repeat,
		url(../assets/images/buildings_background_02.jpg) bottom no-repeat;
```

unless Prettier changes this.